src/sass/_theme_rst.sass

```diff
@@ -520,10 +520,8 @@
     border-radius: $base-line-height / 6
     padding: ($base-line-height / 10) ($base-line-height / 4)
     margin: auto ($base-line-height / 12)
-  *:not(kbd) > kbd,
-  *:not(kbd) > .kbd,
-  *:not(.kbd) > kbd,
-  *:not(.kbd) > .kbd
+  *:not(dl.option-list) > :not(dt):not(kbd):not(.kbd) > kbd,
+  *:not(dl.option-list) > :not(dt):not(kbd):not(.kbd) > .kbd
```

Comment:
This will not work for `.rst-content > p > kbd` ?

Reply:
Maybe that doesn't matter since everything is inside a `section`

Reply:
Okay there are so many nested elements that `.rst-content > p > kbd` seems completely irrelevant

Reply:
Yeah, it seems to work fine here at least:

![image](https://user-images.githubusercontent.com/1140183/186517116-f9c55d23-2024-4f42-b890-cdd74be64b83.png)

Without two/three levels of descendants between `.rst-content` and `kbd`, it could not match though. I can't say for certain that is a potential output, but there should be enough elements in between to avoid that scenario.